Doc/conf.py

```diff
@@ -635,13 +635,14 @@
     'image': '_static/og-image.png',
     'line_color': '#3776ab',
 }
-ogp_custom_meta_tags = [
-    '<meta name="theme-color" content="#3776ab">',
-]
-if 'create-social-cards' not in tags:  # noqa: F821
-    # Define a static preview image when not creating social cards
-    ogp_image = '_static/og-image.png'
-    ogp_custom_meta_tags += [
-        '<meta property="og:image:width" content="200">',
-        '<meta property="og:image:height" content="200">',
+if 'epub' not in tags:  # noqa: F821
```

Comment:
Do we want this HTML for other builders like `latex` and `texinfo`?

Or only for `html`?

```suggestion
if 'html' in tags:  # noqa: F821
```

Reply:
```suggestion
if 'builder_html' in tags:  # noqa: F821
```
It makes sense to limit it only to HTML and skip for latex. Though the format of [several other builders](https://www.sphinx-doc.org/en/master/usage/builders/) is `html`. To make it work explicitly, we'd need to use `builder_html` tag.